hooks/validators/deny_host_ports.go

```diff
@@ -0,0 +1,35 @@
+package validators
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// DenyHostPorts is a Validator that denies usage of HostPorts
+func DenyHostPorts(ctx context.Context, pod *corev1.Pod) field.ErrorList {
```

Comment:
The host port field in PSP accepts a range of ports to be allowed.
I'm not sure this is needed right now, but please consider it is needed or not.